libraries/Microsoft.Bot.Connector/Authentication/SkillValidation.cs

```diff
@@ -23,7 +23,7 @@ public class SkillValidation
         private static readonly TokenValidationParameters _tokenValidationParameters =
             new TokenValidationParameters
             {
-                ValidateIssuer = true,
+                ValidateIssuer = false,
```

Comment:
Why false ?

Reply:
We can validate issuers, but "https://sts.windows.net/d6d49420-f39b-4df7-a1dc-d59a935871db/ below is a hard-coded tenant id, so that's not going to work :)

Reply:
There is a version of TokenValidationParameters that takes a delegate, so you can make sure the issuer is AAD (starts with the aad url"), without hard-coding the tenant id. Or, caller can pass the tenant id in.